modules/appeals_api/app/controllers/appeals_api/v1/appeals_controller.rb

```diff
@@ -28,27 +28,21 @@ def required_header(name)
       end
 
       def ssn
-        @ssn ||= icn_to_ssn!(params[:icn])
-      end
-
-      def va_user
-        required_header('X-VA-User')
+        @ssn ||= icn_to_ssn!(veteran_icn)
       end
 
       def caseflow_request_headers
-        { 'Consumer' => request.headers['X-Consumer-Username'], 'VA-User' => required_header('X-VA-User') }
+        { 'Consumer' => request.headers['X-Consumer-Username'] }
```

Comment:
I was able to dig up some information on the history of `VA-User` and `X-VA-User`. [Here's a GitHub issue that explains the addition of the header (completed in 2018)](https://github.com/department-of-veterans-affairs/vets-contrib/issues/169), and [the PR where the header was added](https://github.com/department-of-veterans-affairs/vets-api/pull/2024). It mentions that the `VA-User` will be logged to CorpDB by Caseflow. So I think we should follow up with Caseflow as soon as we can to ask if that functionality is still needed.

Reply:
Thanks, I've reverted the `X-VA-User` changes for now, and I'll just file a new ticket to clarify `VA-User` for us. Even if we don't change anything, it would help to get more info and update our docs.